src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import type {ElementRef} from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import checkboxStyles from './checkbox.scss';
+
+type Props = {
+    checked: boolean,
```

Comment:
Should we handle in the view component not the same way as the browser does it. So return always a string(default: `on`) or null.